common/recordings.md

```diff
@@ -98,3 +99,39 @@ You can specify a cloud storage location for your recording files in your templa
 ![Destinations](/docs/docs/v2/recording-storage-settings-step6.png)
 
 The above message ensures that your configuration is successful now, and all your recordings will start collecting in your configured destination.
+
+## Storage path for recordings
```

Comment:
most users will rely on using a webhook to figure this out, because predicting the path is not easy (since it contains non-guessable strings like epoch timestamp and stream id). we're confusing users if we tell them about these paths without giving an easy way to get these paths (through webhooks). can we add that content here?